Person.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace GetDisplay
+
+{
+    class Person
+    {
+        public string Name { get; set; }
+        public int FavoriteNumber { get; set; }
+        public int NumberOfKids { get; set; }
+        public string FavoriteAnimal { get; set; }
+
+
+
+        public string AnswerQuestion(string question)
+        {
+            Console.WriteLine($"Hey {Name}, I have a question for you:");
+            Console.WriteLine(question);
+            return Console.ReadLine();
+        }
+
+        public int AnswerQuestionWithInteger(string question)
+        {
+            string stringAnswer = AnswerQuestion(question);
+            return int.Parse(stringAnswer);
+        }
+
+        public int FavoriteNumberAnswer(string question)
+        {
+            string stringAnswer = AnswerQuestion(question);
+            if (int.Parse(stringAnswer) <= 5)
```

Comment:
`int.Parse()` is on 5 different lines.  This should probably be done once and stored in a variable prior to the if-statement, then use the variable in those 5 places